### PR TITLE
Master

### PR DESCRIPTION
**Pull request by wowalid**
Creation mondeV1

### CODE REVIEW

**Comment by massimotisi** [comment]
I need the OK from all the @CACAO2017/lieutenants  to switch to MondeV1 from distributeur.europe

**Comment by massimotisi** [comment]
There is an error in MondeV1.java (lines 23-24: The method ajouterActeur(Acteur) in the type Monde is not applicable for the arguments (Producteur))

File: CACAO2017/src/abstraction/distributeur/europe/Chocolats.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package abstraction.distributeur.europe;
+
+public class Chocolats {
+		private String chocolat;
+		public Chocolats(String chocolat){
+			if ((chocolat == "chocolat_au_lait") || chocolat == "chocolat_noir" || chocolat == "chocolat_blanc"){
```
Comment (duchuyctlk) [inline CACAO2017/src/abstraction/distributeur/europe/Chocolats.java:6]: IMO `(chocolat == "chocolat_au_lait" || chocolat == "chocolat_noir" || chocolat == "chocolat_blanc")` looks a bit mỏe simple 😄

File: CACAO2017/src/abstraction/distributeur/europe/Stock.java [hunk 2]
```diff
@@ -1,39 +1,61 @@
...
 	
 	public void deleteElement(Chocolats Choco, int nombre_de_choco){
-		for (int i=0; i<nombre_de_choco; i++){
-			this.getStock().remove(Choco);
+			this.getStock().remove(new QuantiteChoco(Choco, nombre_de_choco));
+	}
+	
+	public int nbChoco(){
+		int nb = 0;
+		for (int i=0; i<this.getStock().size(); i++){
```
Comment (duchuyctlk) [inline CACAO2017/src/abstraction/distributeur/europe/Stock.java:55]: Why don''t you use `this.stock` directly?

**Comment by antoineroson** [comment]
OK to switch it from prod cotedivoire! @massimotisi

**Comment by antoineroson** [comment]
OK from romuald also !

**Comment by danjud** [comment]
ok to switch from prod US

**Comment by aprage16** [comment]
Ok from transformateur usa





De : antoineroson
Envoyé : ‎vendredi‎ ‎5‎ ‎mai‎ ‎2017 ‎16‎:‎28
À : CACAO2017/CACAO2017
Cc : aprage16, Team mention





OK to switch it from prod cotedivoire! @massimotisi

**Comment by theostourm** [comment]
Ok from transformateur europe

**Comment by AnaisBel** [comment]
Ok from producteur Amerique Latine

**Pull request merged**